apps/src/templates/TeacherFeedback.jsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/no-danger */
+
+import React, {PropTypes, Component} from 'react';
+import { connect } from 'react-redux';
+import color from "../util/color";
+import i18n from '@cdo/locale';
+import { ViewType } from '@cdo/apps/code-studio/viewAsRedux';
+
+const styles = {
+  container: {
+    margin: 20,
+    borderWidth: 5,
+    borderStyle: 'solid',
+    borderColor: color.cyan,
+    backgroundColor: color.lightest_cyan,
+    borderRadius: 5
+  },
+  header: {
+    color: color.white,
+    backgroundColor: color.cyan,
+    padding: 5,
+    fontSize: 18,
+    fontFamily: '"Gotham 7r", sans-serif'
+  },
+  content: {
+    padding: 10
+  }
+};
+
+class TeacherFeedback extends Component {
+  static propTypes = {
+    teacherView: PropTypes.bool
+  };
+
+  render() {
+    if (!this.props.teacherView) {
```

Comment:
I'd pass in "viewAs" as a prop directly and then do the check for `ViewType.Teacher` here because it's more consistent with other code I've seen us do. But if this is how we've done it in other labs places that's good too.